.github/workflows/ci.yaml

```diff
@@ -22,12 +22,11 @@ jobs:
             - name: Build and run tests
               env:
                   PDO_INTERPRETER: ${{ matrix.interpreter }}
-                  PDO_LEDGER_TYPE: ${{ matrix.pdo_ledger_type }}
               run: |
```

Comment:
Yes, it does.
But since we have a single ledger now, and the idea is to avoid ledger-specific suffixes in the makefile, I have removed it.
For testing, however, we may restore this in the future.